src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/model/party/Token.scala

```diff
@@ -21,11 +19,11 @@ import scala.util.Try
 final case class PartyRelationshipBinding(partyId: UUID, relationshipId: UUID)
 
 final case class Token(
-  id: String,
-  legals: Seq[PartyRelationshipBinding],
-  validity: OffsetDateTime,
+  id: UUID,
   checksum: String,
-  seed: UUID
+  applicationId: String,
```

Comment:
what application is in our domain? Is it the product, a reference to current consumer client id?

Reply:
[Here](https://github.com/pagopa/pdnd-interop-uservice-party-management/blob/2004dd839ec9d922ecbd098b02a0671e017998af/src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/model/party/PersistedPartyRelationship.scala#L25) the explanation of what is `applicationId`.

I think it is not necessary.